test/examples/src/man/tutorial.md

```diff
@@ -66,12 +66,7 @@ julia> a / b
 
 ```@eval
 import Markdown
-code = string(sprint(Base.banner), "julia>")
-Markdown.MD([Markdown.Code(code)])
-```
-
-```@eval
-rand(20, 20)
+Markdown.MD("Hello")
```

Comment:
`Base.banner` was moved to `REPL.banner` in Julia 1.11. But anyway, there seems no point in having a complex test here, is there?